cli/main.rs

```diff
@@ -1068,14 +1077,14 @@ async fn test_command(
   if flags.watch {
     tools::test::run_tests_with_watch(
       flags,
-      include,
-      ignore,
-      doc,
-      no_run,
-      fail_fast,
-      filter,
-      shuffle,
-      concurrent_jobs,
+      test_flags.include,
```

Comment:
Nice, this is pretty much exactly what I had in mind.
I'll follow up with a cleanup of these signatures.